libs/ledgerjs/packages/context-module/src/token/data/HttpTokenDataSource.ts

```diff
@@ -8,13 +8,52 @@ export class HttpTokenDataSource implements TokenDataSource {
     chainId,
     address,
   }: GetTokenInfosParams): Promise<string | undefined> {
-    const response = await axios.request<TokenDto[]>({
```

Comment:
You should use `TokenDto` here

Reply:
Yup, wasn't there at the time, I just rebased and missed the change, but will do 👍